### ISSUE TITLE
Spaces in Romanian

### ISSUE THREAD

**Issue by domian** [clics/clics1#11]
many (all?) Romanian verbs have the shape a (verb), e.g. a bate 'strike'. In CLICS we have abate instead, i.e. the space has been deleted. Have all spaces been removed automatically during conversion in all datasets? This may be far too broad-sweeping...

**Comment by LinguList** [comment]
yep, @cysouw gave me a list of regexes he used for conversion, including the space deletion, but we probably should reconsider using it. So le't say that in the next version, no space deletion will occur.
